src/App.tsx

```diff
@@ -1,7 +1,11 @@
-import { OnboardingPage } from './pages/OnboardingPage';
+import { Outlet } from '@tanstack/react-router';
 
 function App() {
-    return <OnboardingPage />;
+    return (
+        <>
```

Comment:
No need for fragment, fragment should only be used if more than one component is rendered on the same level

Reply:
There will be a navigation and a footer. That's why i left the fragment in the code